spaces/S000173/properties/P000200.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000173
+property: P000200
+value: true
+refs:
+  - zb: "1044.55001"
+    name: Algebraic Topology (Hatcher)
+---
+
+Note that a product is path-connected if and only if its factors are. In addition to this, the fundamental group factors through products, so the same is true of simply-connectedness; see Proposition 1.12 of {{zb: "1044.55001"}}. Since {S38|P200} and {S103|P200}, it follows that $X$ is simply connected.
```

Comment:
```suggestion
A product is path-connected if its factors are. In addition, the fundamental group factors through products, so the same is true of simply-connectedness; see Proposition 1.12 of {{zb: "1044.55001"}}. Since {S38|P200} and {S103|P200}, it follows that $X$ is simply connected.
```
I suggest modification of the first sentence, because as written it was not quite correct. I know it's pedantic, but also trivial to express correctly.  The iff was only valid if all factors are nonempty.  So we could have said "a nonempty product is path-connected if and only if ...".  But it's even easier to say what is suggested, as we only need the "if" part.  (The purpose of these proofs is not to "educate" the reader, it's to get an argument that is clear, short and to the point.)